### PR TITLE
1.9.0 release

### PR DESCRIPTION
**Pull request by carohadad**

### CODE REVIEW

File: docs/whats-new.md [hunk 1]
```diff
@@ -44,13 +44,17 @@ If you prefer sharing your files through another app:
...
+- Added support for the relationship property type in the Uwazi connection.
+- Added offline support for the Uwazi server connection (Drafts, Outbox and Submitted tabs). More details [here](/uwazi).
+-  Bug fixing and small improvements:
+  - Run all remaining database migrations on every update. This bug was causing an inconsistent database state for users who don't use Tella often.
+  - Fix the strip EXIF metadata feature that had been broken since the last database migration.
+  - Show a warning if the users tries to delete a file that's being used in server connection that's not submitted.
+  - Hide password field in a new Tella Web server connection.
+  - Make the select field on Uwazi connections scrollable.
+  - Fix back button issues on the onboarding flow.
+  - Fix the import functionality when no permissions has been granted.
```
Comment (dhekra-rouatbi) [inline docs/whats-new.md:58]: maybe we can also add :
`The user can select multiple files from the Photo Library.`

**Review by dhekra-rouatbi — CHANGES_REQUESTED** [review]

File: docs/whats-new.md (hunk 1 above)
Comment (carohadad) [reply docs/whats-new.md:58]: nice! for import you mean? Is it there a task for that in asana that  I can refer to?
If there isn't, tell me and also let's bring it to co-design just to make sure Figma is update and android has a similar task as well

Comment (carohadad) [reply docs/whats-new.md:58]: cc @dhekra-rouatbi

Comment (dhekra-rouatbi) [reply docs/whats-new.md:58]: @carohadad I haven't created a task for that but it was fixed by the Exiff feature. Yes we can discuss that in the co-design

Comment (carohadad) [reply docs/whats-new.md:58]: ok then ! I won't put it in the release notes, so it's a hidden feature for now. just top of mind, we need to double check how the "delete original" feature should work (meaning, delete all!) and maybe update the text for the confirmation box. And make sure android behaves similrarly

**Review by raphmim — APPROVED** [review]
Looks awesome :) Amazing to see so much progress 💙💙💙 cc @carohadad @dhekra-rouatbi @ValbuenaG @Rimktarii

**Review by dhekra-rouatbi — APPROVED** [review]

**Comment by carohadad** [comment]
@raphmim  i need to merge this so i can deploy to beta . In case you want to change anything, still tell me and i will fix it in other pr.

**Pull request merged**